jwebunit-htmlunit-plugin/src/main/java/net/sourceforge/jwebunit/htmlunit/HtmlUnitTestingEngineImpl.java

```diff
@@ -764,9 +760,6 @@ public String getPageText() {
     if (page instanceof TextPage) {
       return ((TextPage) page).getContent();
     }
-    if (page instanceof JavaScriptPage) {

```

Comment:
JavaScriptPage was removed from htmlunit and according to the commit message, it should be treated the same way as HtmlPage